src/includes/getting-started-primer/dotnet.mdx

```diff
@@ -0,0 +1,11 @@
+Our .NET SDK supports all recent versions of the platform, and integrates well with a variety of popular frameworks and packages in the ecosystem. It gives developers helpful hints for where and why an error or performance issue might have occurred.
+
+The main [Sentry NuGet package](https://www.nuget.org/packages/Sentry) targets .NET Standard 2.0. [Our legacy SDK](/clients/csharp/) supports .NET Framework as early as 3.5. Read more about [compatibility and unit testing](./unit-testing.mdx).
```

Comment:
I think here we could say that:

> The lowest version this SDK is compatible with are .NET Standard 2.0 and .NET Framework 4.6.1.
> For older versions, such as .NET Framework 3.5, please refer to our Raven SDK.

The reference about Unit Testing here isn't a good fit I believe. Also something we can just move to configuration somehow.